.taskcluster.yml

```
@@ -220,7 +220,7 @@ tasks:
           provisionerId: proj-bugbug
           workerType: compute-large
           payload:
-            maxRunTime: 10800
+            maxRunTime: 2700
```

Based on @marco-c's suggestion in https://github.com/mozilla/bugbug/pull/3810#issuecomment-1804800058, we could keep the current value as the default value and customize it for the models that need less time. For now, we could start with the  `regression` model. We will later add more models as required.

```suggestion
            maxRunTime:
              {
                $switch:
                  {
                    '"Train on Taskcluster: regression" in pr_description': 2700,
                    $default: 10800,
                  },
              }
```